pfsenseapi/firewall.go

```diff
@@ -272,20 +287,30 @@ type firewallRuleCreateRequest struct {
 }
 
 // CreateRule creates a new Rule
-func (s FirewallService) CreateRule(ctx context.Context, newRule FirewallRuleRequest, apply bool) error {
+func (s FirewallService) CreateRule(ctx context.Context, newRule FirewallRuleRequest, apply bool) (int, error) {
```

Comment:
I haven't tried this endpoint, and the API docs do not have the responses documented.

Does the response from the API contain the entire firewall rule object that was created? I would assume so. If this is the case, rather than returning just the tracker ID here, can we return the entire`*FirewallRule`?